common/religion/religions/shamanism_group.txt

```diff
@@ -650,11 +650,8 @@
 
 			doctrine = unreformed_faith_doctrine
 			
-			holy_site = icemist #These are placeholders, can't put proper ones until Broken Isles are mapped
-			holy_site = taunkale
-			holy_site = wintergarde
-			holy_site = njord
-			holy_site = grizzlemaw
+			holy_site = neltharions_lair # Origin + thunder totem
+			holy_site = rockcrawler_chasm
```

Comment:
They should have at least 5 holy sites

Reply:
which ones do you suggest

Reply:
The Highmountain Capital maybe?

Reply:
And the Suramar, Zarkhenar and the Azurewing nest since they are close?

Reply:
neltharions is already at thunder totem but ill add the rest